lib/airbrake/rake/tasks.rb

```diff
@@ -106,8 +106,19 @@
            " environment will be used."
     end
 
+    unless (repo = ENV['REPOSITORY_URL'])
+      repo = `git remote get-url origin 2>/dev/null`.chomp
+      if repo.empty?
+        puts "Airbrake couldn't identify your app's repository."
```

Comment:
I'll add a PR to update the documentation. The heroku deploy docs are in need of an update anyway, so I'll include this change along with the rest.